qubes/ext/pci.py

```diff
@@ -19,10 +19,12 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+''' Qubes PCI Extensions '''
+
 import os
 import re
 import subprocess
-import libvirt
+import libvirt  # pylint:disable=import-error
```

Comment:
Take a look at pylint call in `.travis.yml`. There is mock `libvirt` module in `test-packages` directory exactly for this purpose. Anyway this pylint disable can be left here.